libraries/Microsoft.Bot.Builder/SlotDefinition.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder
+{
+    public class SlotDefinition
+    {
+        public string Name { get; set; }
+
+        public object DefaultValue { get; set; }
+
+        public string[] ChangeTags { get; set; }
+
+        public int ExpiresAfterSeconds { get; set; }
```

Comment:
Why is this here if this is on `SlotHistoryPolicy`? Seems like it should only be in one place.